### PR TITLE
feat(terms and conditions): fixed rendering

### PR DESCRIPTION
**Pull request by milan-bc**
## Description (optional)
Add a concise explanation of the changes.

## Testing Steps (optional)
Detail the steps required for the reviewer(s) to verify and test these changes.

### CODE REVIEW

File: packages/blockchain-wallet-v4-frontend/src/components/Flyout/TermsAndConditions/index.tsx [hunk 1]
```diff
@@ -126,22 +117,13 @@ const TermsAndConditions: React.FC<InjectedFormProps<{}, Props> & Props> = (prop
...
-                    component: StyledParagraph
-                  }
-                }
-              }}
-            >
-              {props.termsAndConditions.termsAndConditions}
-            </Markdown>
+            <TextWrapper>
+              <TextBox
+                dangerouslySetInnerHTML={{ __html: props.termsAndConditions.termsAndConditions }}
```
Comment (blockdylanb) [inline packages/blockchain-wallet-v4-frontend/src/components/Flyout/TermsAndConditions/index.tsx:124]: I know this data is coming from our own endpoint so there's minimal risk here but just to be safe think we should wrap this in DOM purify https://www.npmjs.com/package/dompurify

**Review by blockdylanb — APPROVED** [review]
.

**Review by blockdylanb — CHANGES_REQUESTED** [review]
we should implement DOM purify on the dangerous html section.

**Comment by milan-bc** [comment]
Let's wait until we are sure about HTML vs markdown

File: packages/blockchain-wallet-v4-frontend/src/components/Flyout/TermsAndConditions/index.tsx [hunk 2]
```diff
@@ -126,22 +118,15 @@ const TermsAndConditions: React.FC<InjectedFormProps<{}, Props> & Props> = (prop
...
-                  }
-                }
-              }}
-            >
-              {props.termsAndConditions.termsAndConditions}
-            </Markdown>
+            <TextWrapper>
+              <TextBox
+                dangerouslySetInnerHTML={{
+                  __html: DOMPurify.sanitize(props.termsAndConditions.termsAndConditions)
```
Comment (blockdylanb) [inline packages/blockchain-wallet-v4-frontend/src/components/Flyout/TermsAndConditions/index.tsx:126]: 👏🏻

**Review by blockdylanb — APPROVED** [review]

**Pull request merged**